satellite/satellitedb/storagenodeaccounting.go

```diff
@@ -195,18 +195,23 @@ func (db *StoragenodeAccounting) QueryPaymentInfo(ctx context.Context, start tim
 	return csv, nil
 }
 
-// QueryNodeDailySpaceUsage returns slice of NodeSpaceUsage for given period
-func (db *StoragenodeAccounting) QueryNodeDailySpaceUsage(ctx context.Context, nodeID storj.NodeID, start time.Time, end time.Time) (_ []accounting.NodeSpaceUsage, err error) {
+// QueryNodeStorageUsage returns slice of NodeStorageUsage for given period
+func (db *StoragenodeAccounting) QueryNodeStorageUsage(ctx context.Context, nodeID storj.NodeID, start time.Time, end time.Time) (_ []accounting.NodeStorageUsage, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	// as entries are stored on daily basis we don't need
-	// to extract DATE from start_time
-	query := `SELECT at_rest_total, start_time 
+	query := `SELECT id, at_rest_total, start_time, 
```

Comment:
this query needs to change if id is removed.

Reply:
`ID` is no longer a primary key here. It is stored to be able to reference appropriate `rollup` entry if needed

Reply:
I think we should remove the ID here and in the protobuf as mentioned above.
I cannot find a line where we use the ID, so it does not have any benefit (yet).

Reply:
I was wondering if by anyway you have two rollups, for the same time, and resulting one has wrong data, then we can use this id to reference where it came from. But yes, it is not used anywhere now. So I am going to change that